GitCommands/GitCommands.csproj

```diff
@@ -35,6 +35,7 @@
     <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
     <TargetFrameworkProfile>
     </TargetFrameworkProfile>
+    <LangVersion>7.2</LangVersion>
```

Comment:
:memo: This should be set in a **Directory.Build.props** file that gets used by all projects automatically.